services/payments/tests/unit/test_rpc_payments.py

```diff
@@ -54,7 +57,7 @@ def app_environment(
 @pytest.fixture
 def init_payment_kwargs(faker: Faker) -> dict[str, Any]:
     return {
-        "amount_dollars": 100,
+        "amount_dollars": 999999.99609375,  # SEE https://github.com/ITISFoundation/appmotion-exchange/issues/2
```

Comment:
So here you are testing more or less the upper bound of what payment we allow?

Reply:
If you follow the link I put in there you will see the actual issue :-) 
This is an edge case found by another separate test system that explores different values based on the opeanpi specs. SEe schemathesis